02-install/02-01-install_rust_on_linux.md

```diff
@@ -1,50 +1,52 @@
 
-查看操作系统请在终端执行如下命令:
+To view the operating system, execute the following command in the terminal:
 
 `uname -a`
 
-结果如下图所示：
+The result is shown below：
 
 ![check system info](../image/01-install-on-linux-check-system.png)
 
-如上图所示，如果是 **x86_64** 则证明是64位系统，需要[下载](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64位安装包；
+As shown above, if it is **x86_64** 64-bit system，to [download](https://static.rust-lang.org/dist/rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz)64-bit installation package；
 
-如果是**x86-32**则需要[下载](https://static.rust-lang.org/dist/rust-1.5.0-i686-unknown-linux-gnu.tar.gz)32位安装包
+If operating system is **x86-32** to [download](https://static.rust-lang.org/dist/rust-1.5.0-i686-unknown-linux-gnu.tar.gz)32-bit installation package;
 
-下载安装包后解压运行即可。*在书写本章时，最新的稳定版本为1.5.0，*
- 解压：`tar -zxvf rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz`
+After downloading the installation package, unzip and run. *At the time of this writing, the latest stable version is 1.5.0,*
+ Unzip：`tar -zxvf rust-1.5.0-x86_64-unknown-linux-gnu.tar.gz`
  
- 解压完进入对应的目录：`cd rust-1.5.0-x86_64-unknown-linux-gnu`
- 执行 `./install.sh`
+After decompression, enter the corresponding directory：`cd rust-1.5.0-x86_64-unknown-linux-gnu`
+ 执行: `./install.sh`
  
-上述命令执行完成后会打印： **Rust is ready to roll.** 表明安装成功
+After the above command is executed, it will print： **Rust is ready to roll.** This indicates successful installation.
 
-此时执行: `rustc --version`, 你会看到对应的 rust 版本信息,如下图所示：
+Execute at this time: `rustc --version`, You will see the corresponding rust version information, as shown below：
 
 ![Success and show rust version info](../image/01-install-on-linux-rust-success.png)
 
-### 2、命令行一键安装：
-Rust 提供简单的一键安装，命令如下：
+### 2、One-click installation from the command line：
+Rust provide simple one-click installation, the command is as follows：
 
 `curl -sSf https://static.rust-lang.org/rustup.sh | sh`
 
-打开终端执行如上命令即可。
 
+Open the terminal and execute the above command.
 
-**注意**
 
-除了稳定版之外，Rust 还提供了 Beta 和 Nightly 版本，下载地址如下：
+**note**
+
+
+In addition to the stable version，rust also provided Beta with Nightly version，download address is as follows：
 https://www.rust-lang.org/downloads.html
 
-如果你不想安装 Rust 在你的电脑上，但是你还是像尝试一下 rust，那么这里有一个在线的环境：http://play.rust-lang.org/
+If you don't want to install rust on your computer，but you still want try rust，Then here is an online environment：http://play.rust-lang.org/
 
-中国科学技术大学镜像源包含 [rust-static](http://mirrors.ustc.edu.cn/rust-static/)，梯子暂时出问题的同学可以尝试从这里下载编译器；除此之外。还有 Crates 源，详见[这里的说明](https://servers.ustclug.org/2016/01/mirrors-add-rust-crates/)。
+University of Science and Technology of China mirror source contains [rust-static](http://mirrors.ustc.edu.cn/rust-static/)，Students who have problems with the ladder temporarily can try to download the compiler from here; and also Crates source，See details[Instructions here](https://servers.ustclug.org/2016/01/mirrors-add-rust-crates/)。
```

Comment:
This is Chinese mainland's mirror, I think that some one not live in there may never use that. You could delete permanently.